src/local/butler/deploy.py

```diff
@@ -236,6 +256,25 @@ def _update_bigquery(project):
                              os.path.join('bigquery', 'datasets.yaml'))
 
 
+def _update_redis(project):
+  """Update redis instance."""
+  _update_deployment_manager(project, 'redis',
+                             os.path.join('redis', 'instance.yaml'))
+
+  region = appengine.region(project)
+  return_code, _ = common.execute(
+      'gcloud compute networks vpc-access connectors describe '
+      'connector --region={region} '
+      '--project={project}'.format(project=project, region=region),
+      exit_on_error=False)
+
+  if return_code != 0:
```

Comment:
nit: Add a comment that this means it is already created.

Reply:
Done!